indico/modules/events/registration/util.py

```diff
@@ -187,6 +187,8 @@ def create_personal_data_fields(regform):
             continue
         field = RegistrationFormPersonalDataField(registration_form=regform, personal_data_type=pd_type,
                                                   is_required=pd_type.is_required)
+        if not data.get('is_enabled', True):
+            field.position = data.get('position')
```

Comment:
Assigning a `None` positions seems odd. Do we ever expect `position` to be missing? If not let's use `data['position']`